template/styles/dashboard.css

```
@@ -0,0 +1,333 @@
+body{
+  font-family: 'Montserrat', sans-serif;
+  background-color: 	#f9fff9;
+}
+body, html{
+  background-color: rgb(248, 244, 244);
+}
+.container{
+  display: flex;
+}
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px;
+  background-color: #fff;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 60px;
+  color: #fff;
+  z-index: 999
+    
+}
+
+.navbar__left {
+  display: flex;
+  align-items: center;
+  margin-left: 25px;
+}
+.navbar__text{
+  color: black;
+}
+.navbar__menu {
+  cursor: pointer;
+  padding-right: 10px;
+}
+
+.navbar__left-logo img{
+  width:60px;
+  height: 60px;
+  padding:0px 50px;
+  cursor: pointer;
+}
+
+
+.navbar__right{
+  padding:10px 30px;
+  
+}
+.navbar__right img {
+  display: flex;
+  align-items: center;
+  height:50px;
+  width: 50px;
+  object-fit: cover;
+  border: 1px solid red;
+  border-radius: 10px;
+  cursor: pointer;
+}
+
+.sidebar {
+  position: fixed;
+  left: 0;
+  top: 100px;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  margin-top: -30px;
+  width: 250px;
+  padding: 10px;
+  background-color: white;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+}
+.sidebar__card{
+  background-color: rgb(233, 224, 224);
+  height: 100px;
+  margin-top: 10px;
+  border-radius:10px;
+  margin-bottom:20px;
+  width:100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+}
+.sidebar__card img{
+  height:60px;
+  width: 60px;
+  border-radius:10px;
+  object-fit: cover;  
+}
+.sidebar__container{
+  margin-left:10px;
+}
+
+ .owner-section__name {
+  margin: 0;
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.owner-section__role {
+  margin: 0;
+  color: #888;
+}  */
```

This must be a typo, remove it.

```suggestion
} 
```